app/gui/src/controller/searcher/component/group.rs

```diff
@@ -9,11 +9,34 @@ use crate::controller::searcher::component::MatchInfo;
 use crate::model::execution_context;
 use crate::model::suggestion_database;
 
+use double_representation::project;
 use ensogl::data::color;
 use std::cmp;
 
 
 
+// =====================
+// === QualifiedName ===
+// =====================
+
+/// Fully qualified name of a component group.
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct QualifiedName {
+    /// Fully qualified name of the project containing the definition of the group.
+    pub project: project::QualifiedName,
+    /// The name of the group in the scope of the project where it was defined.
+    pub name:    ImString,
+}
+
+impl QualifiedName {
+    /// Construct.
+    pub fn new(project: project::QualifiedName, name: impl Into<ImString>) -> Self {
+        Self { project, name: name.into() }
+    }
+}
+
+
+
```

Comment:
It is the Qualified name for the virtual groups only, but it cannot be deduced from docs nor from the struct's name and location. It should be generalized, or renamed, or moved to a special module dedicated to virtual groups.

Reply:
Thanks for the comment. I decided to remove the type. It was not really used in many places, and where it was, passing a name of a virtual group plus a `project::QualifiedName` instead seems less confusing.